ui/app/templates/components/get-credentials-card.hbs

```diff
@@ -6,24 +6,32 @@
     <p class="is-label search-label">{{@searchLabel}}</p>
   </div>
   <p class="sub-text">{{@subText}}</p>
-  <SearchSelect
-    @id={{this.id}}
-    @models={{@models}}
-    @selectLimit="1"
-    @backend={{@backend}}
-    @fallbackComponent="input-search"
-    @shouldUseFallback={{@shouldUseFallback}}
```

Comment:
having `shouldUseFallback` as an arg _and_ tracked variable set by the component was confusing (it was set to `true` if the query returned a `403`)

Reply:
I think we should keep the `403` functionality. The use case there is that if someone is filling a form but doesn't have the permissions to see the list, they can still fill out the form correctly as long as they know the name of the item they need for that input. 
That being said, if it's being set by the parent it doesn't really make sense to force the child (SearchSelect) to render, since the parent already knows we don't really want the SearchSelect, so the pattern you have here makes sense to extend. All in all, I think `shouldUseFallback` in SearchSelect should be a component attribute rather than a passed attribute, and we'll leave it to the parents who know they want to use the fallback to just render the desired fallback instead